images/virtualization-artifact/pkg/controller/vmip/vmip_reconciler.go

```diff
@@ -73,7 +73,7 @@ func (r *Reconciler) SetupController(_ context.Context, mgr manager.Manager, ctr
 		source.Kind(mgr.GetCache(), &virtv2.VirtualMachine{}),
 		handler.EnqueueRequestsFromMapFunc(r.enqueueRequestsFromVMs),
 		predicate.Funcs{
-			CreateFunc: func(e event.CreateEvent) bool { return false },
+			CreateFunc: func(e event.CreateEvent) bool { return true },
```

Comment:
No need to watch for VirtualMachine create, "false" is totally OK here.

You need to return true from UpdateFunc on line 66 — in VirtualMachineIPAddressLease watcher. Or use DeepEqual on fields of interest to narrow reconciles.

Reply:
Oh, sorry, a typo. Fixed it.